resources/troubleshooting-guides/push_notification_troubleshooting.md

```diff
@@ -0,0 +1,97 @@
+{% endtab %} {% endtabs %}
+
+{% tabs %} { % tab title="Android" % }
+
+  * **Verify that Server key is configured in Campaign instance.**
+
+    Ensure that server key is successfully set up in Campaign instance.  
+    This can be verified in Campaign instance under following section Campaign -> Administrator -> Channels -> Mobile App (AEP SDK).  
+    Launch mobile app and check push channel settings, _it should say Android key set up success_. It should look like:
+    
+   ![alt text](https://github.com/shivam-tomar-sde/aep-sdks-documentation/blob/push-troubleshooting-document/.gitbook/assets/android_server_key.png "Server key configuration in Android.")
+
+{% endtab %}
+
+{% tabs %} { % tab title="iOS" % }
+
+  * **Verify that Server cerificate is configured in Campaign instance.**
+  
+      Ensure that server certificate (.pem file) is successfully set up in Campaign instance. This can be verified in Campaign instance under following section Campaign -> Administrator -> Channels -> Mobile App (AEP SDK). Launch mobile app and check push channel settings, it should say Android key set up success. It should look like:
+  ![alt text](https://github.com/shivam-tomar-sde/aep-sdks-documentation/blob/push-troubleshooting-document/.gitbook/assets/campaign_ios_cert_configured.png "Configured push certificate in Campaign.")
+
+{% endtab %}
+
+{% endtabs %}
+
+{% tabs %} { % tab title="Android" % }   
+  * **Set up Android app for receiving push notifications: Android app should contain following for receiving push notifications:**
+
+    * Create Firebase Messing Service and add it to Android Manifest file. [Refer](https://firebase.google.com/docs/cloud-messaging/android/client).
+  
+    * Verify that user is opted in for receiving push notification. This can be verified in Griffon. [Refer](https://aep-sdks.gitbook.io/docs/resources/troubleshooting-guides/troubleshooting-push#ensure-user-opt-in-for-push-in-adobe-analytics).
+  
+    * Generate push token for app using Fire Base Instance Id class. Set push token to AEP SDK by calling Set [Push Identifier](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference#set-up-push-messaging). Verify that Push token is successfully synced with Identity service in Griffon. [Refer](https://aep-sdks.gitbook.io/docs/resources/troubleshooting-guides/troubleshooting-push#verify-push-token-sync-with-the-experience-cloud-identity-service).
+
+    * For app to receive push notification it’s push token should be present in Campaign instance mapped with correct Experience cloud id(mid). This can be verified by following 2 steps:
+                 
+    * Verification in Griffon: Verify that the user id is correctly set. [Refer](https://aep-sdks.gitbook.io/docs/resources/troubleshooting-guides/troubleshooting-push#confirm-that-the-user-id-is-correctly-set).
```

Comment:
Make this step 7:
"To verify that the user id is correctly set in Project Griffon, see [Confirm that the user ID is correctly set](https://aep-sdks.gitbook.io/docs/resources/troubleshooting-guides/troubleshooting-push#confirm-that-the-user-id-is-correctly-set)."